WordPress/Classes/ViewRelated/NUX/SignupDomainSuggestionViewController.swift

```diff
@@ -0,0 +1,241 @@
+import UIKit
```

Comment:
This isn't for Signup, so this filename is misleading. Maybe `SiteCreationDomainSuggestionViewController` or just `DomainSuggestionViewController`?

Reply:
100% agree with this. I created that file weeks ago and didn't think about it since 😂